docs/extending/classes.md

```diff
@@ -73,6 +75,11 @@ module.exports = (srcPath) => {
   // change them if you need to.
   const damagePercent = 250;
   const energyCost = 20;
+  
+  //An example damage calculation using the Combat library from the ranvier-combat bundle
```

Comment:
for the sake of consistency in documentation, please place a space after the `//` in comments (this is consistent with airBnB and standardJS styles also)